extensions/blocks/membership-button/edit.jsx

```diff
@@ -327,6 +329,14 @@ class MembershipsButtonEdit extends Component {
 						} ) ) }
 					/>
 				</PanelBody>
+				<PanelBody title={ __( 'Your members.', 'jetpack' ) }>
+					<ExternalLink href={ `https://wordpress.com/earn/memberships/${ this.state.siteSlug }` }>
+						{ __(
+							'You can find your earnings, subscribers and advanced product management here.',
```

Comment:
This ends up being pretty long in the interface:

![image](https://user-images.githubusercontent.com/426388/58314937-d636c180-7e10-11e9-9519-524a4aafab9a.png)

I wonder if we could make that text shorter.

Reply:
Yeah, I wanted to be explicit. I will defer to editorial feedback.